NLog.Windows.Forms/Targets/RichTextBoxTarget.cs

```diff
@@ -882,22 +882,22 @@ private void SendTheMessageToRichTextBox(string logMessage, RichTextBoxRowColori
 
             int startIndex = textBox.TextLength;
             textBox.SelectionStart = startIndex;
-            textBox.SelectionBackColor = GetColorFromString(rule.BackgroundColor, textBox.BackColor);
-            textBox.SelectionColor = GetColorFromString(rule.FontColor, textBox.ForeColor);
+            textBox.SelectionBackColor = GetColorFromString(rule.BackgroundColor.Render(logEvent), textBox.BackColor);
```

Comment:
`RenderLogEvent(rule.BackgroundColor, logEvent)` is better.

Reply:
Bruh, you could have just commented one time, you didn't need to find every instance. Heck since I used Regex to fix it, I didn't need to find every instance. Although I did then need to change a couple back since `RenderLogEvent` isn't available in `RichTextBoxLinkLayoutRenderer` or `RichTextBoxWordColoringRule`.

Reply:
Yes search-and-replace is a much better solution, and much more effiecient than my scribbles.. Thank you for the fixes.

Will try to avoid pointing out the obvious for you next time :)

Reply:
You're fine. It's the kind of thing I might do too.

I'm also curious, why is `RenderLogEvent` preffered?

Reply:
When using `Target.RenderLogEvent()` then it will allow use of reusable-StringBuilder for reduced memory-allocation, and improved performance.

Reply:
> I'm also curious, why is `RenderLogEvent` preffered?

PS: we should really add this to the XML docs

Reply:
--> https://github.com/NLog/NLog/pull/4947